crates/wadm/src/scaler/manager.rs

```diff
@@ -880,6 +900,15 @@ pub(crate) fn compute_component_id(
     }
 }
 
+pub(crate) fn compute_secret_id(
+    model_name: &str,
+    component_id: Option<&String>,
+    component_name: &str,
+) -> String {
+    let name = compute_component_id(model_name, component_id, component_name);
+    format!("secret_{}", name)
```

Comment:
Any thoughts on if this should be `secret_` or `SECRET_`? The uppercase matches our current bucket naming strategy but I'm not sure that we specifically need that

Reply:
I don't have a strong preference here. Uppercase works if we want to stick with that convention

Reply:
@brooksmtownsend I updated it to be `SECRET_` since uppercase made it seem much more like a generated value.

Reply:
Note to myself that I should change the PR in wasmCloud to use the uppercase convention